packages/arkor-local/python/mlx/dataset_prep.py

```
@@ -0,0 +1,254 @@
+    valid = []
+    if not split:
+        # Best effort: reuse an existing validation split when the dataset
+        # ships one. An explicit `split` request disables this (the user
+        # asked for exactly one split).
+        for candidate in ("validation", "valid"):
+            try:
+                valid = load(candidate)
+                log(f"[arkor] using the dataset's {candidate!r} split for eval")
+                break
+            except Exception:
+                continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)